package.json

```diff
@@ -2,23 +2,25 @@
   "name": "string-ts",
   "version": "1.2.0",
   "description": "Strongly-typed string functions.",
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
+  "main": "dist/commonjs/index.js",
+  "module": "dist/es/index.js",
+  "jsnext:main": "dist/es/index.js",
   "author": "Gustavo Guichard <@gugaguichard>",
   "license": "MIT",
   "scripts": {
-    "build": "tsup ./src/index.ts --format esm,cjs --dts",
+    "build": "npm run clean-dist && npm run build:es && npm run build:cjs",
     "dev": "tsup ./src/index.ts --format esm,cjs --watch --dts",
     "lint": "eslint *.ts*",
-    "tsc": "tsc --noEmit",
```

Comment:
I moved this in the tsconfig.json - overwritten by the tsconfig.build.json when building.